README

```diff
@@ -1,7 +1,7 @@
 Bugzilla
 ========
 
-Bugzilla is free and open source web-based bug-tracking software that is
+Bugzilla is a free and open source web-based bug-tracking software that is
```

Comment:
This does not look right. "Software" is an uncountable noun, isn't it?

Reply:
The subject of the matter is Bugzilla in this particular case

Reply:
But the "a" isn't in front of "bugzilla," it's in front of "software."  @CyberShadow is correct, software is an uncountable noun, this was correct as-is.